api/dao/liststorage.py

```diff
@@ -72,7 +72,10 @@ def _create_el(self, _id, payload, exclude_params):
         update = {'$push': {self.list_name: payload} }
         log.debug('query {}'.format(query))
         log.debug('update {}'.format(update))
-        return self.dbc.update_one(query, update)
+        result = self.dbc.update_one(query, update)
+        if result.matched_count < 1:
```

Comment:
This is no great way to determine when an item already exists in the list without pulling the item first. Since we'll 404 when trying to call POST on a list endpoint of a container that does not exist, a `matched_count` of `0` when querying for `_id` *and* `'<list_name>': {'$not': {'$elemMatch': <item_to_be_inserted}}}` should be enough information to determine that:
  1. the container does indeed exist
  2. the item to be inserted exists in the list